wolfssl/wolfcrypt/settings.h

```diff
@@ -209,6 +209,9 @@
 /* Uncomment next line if using RENESAS RX64N */
 /* #define WOLFSSL_RENESAS_RX65N */
 
+/* Uncomment next line if using Solaris OS*/
```

Comment:
A little after this, you should check for

    #if defined(sun) || defined(__sun)
        #if defined(__SVR4) || defined(__svr4__)
            #define WOLFSSL_SOLARIS
        #endif
    #endif

Reply:
And then remove it from the fips-check.sh script.